client/src/c_console.cpp

```diff
@@ -1366,6 +1365,55 @@ int C_BasePrint(const int printlevel, const char* color_code, const std::string&
 	return newStr.length();
 }
 
+FORMAT_PRINTF(1, 2) int STACK_ARGS Printf(const char* format, ...)
```

Comment:
These FORMAT_PRINTF functions no longer exist in protobreak, looks like they accidentally got brought in from stable at some point. They should be deleted.